src/rospkg/os_detect.py

```diff
@@ -126,6 +139,47 @@ def get_codename(self):
             return self.lsb_info[2]
         raise OsNotDetected('called in incorrect OS')
 
+class FdoDetect(OsDetector):
```

Comment:
Rich mentions that `Fdo` is intended to represent freedesktop.org, and isn't specific to Fedora... Since they host the spec for this file, it makes sense. What would you rather see it changed to?